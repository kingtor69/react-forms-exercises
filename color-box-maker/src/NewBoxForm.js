import React, {useState} from 'react';

const NewBoxForm = () => {
    return (
        <div className="NewBoxForm">
            this is a form to make a new box
        </div>
    )
}

export default NewBoxForm;
