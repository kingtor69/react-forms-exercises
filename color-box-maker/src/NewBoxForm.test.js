import { render } from '@testing-library/react';
import NewBoxForm from './NewBoxForm';

it('renders witout ceashing', () => {
  render(<NewBoxForm />);
});

it('renders snapshot', () => {
  const { asFragment } = render(<NewBoxForm />);
  expect(asFragment()).toMatchSnapshot();
});
