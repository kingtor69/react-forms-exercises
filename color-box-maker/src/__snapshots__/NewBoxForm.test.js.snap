// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<DocumentFragment>
  <div
    class="NewBoxForm"
  >
    this is a form to make a new box
  </div>
</DocumentFragment>
`;
